doc/_extensions/zephyr/application.py

```diff
@@ -320,8 +318,18 @@ def _generate_west(self, **kwargs):
         # defaulting to west.
         #
         # For now, this keeps the resulting commands working.
-        content.append('west build -b {}{}{}{}{}{}{}'.
-                       format(board, west_args, snippet_args, shield_args, build_dst, src, cmake_args))
+        content.append(
+            "west build -b {}{}{}{}{}{}{}{}".format(
+                board,
+                build_args,
+                west_args,
+                snippet_args,
+                shield_args,
+                build_dst,
+                src,
+                cmake_args,
+            )
+        )
```

Comment:
Not sure what the recommended way is, but I tend to prefer formatted string literals.

```suggestion
        content.append(
            f"west build -b {board}{build_args}{west_args}{snippet_args}"
            f"{shield_args}{build_dist}{src}{cmake_args}"
        )
```

Reply:
Sure, why not. :-) Fixed.